pages/Product/components/Reviews/components/Header/components/AverageRating/index.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import RatingStars from '../../../../../../../../components/RatingStars';
+import RatingCount from '../RatingCount';
+import style from './style';
+
+/**
+ * The average rating and number of ratings for a product
+ * @param {Object} rating The rating values
+ * @returns {XML}
+ * @constructor
+ */
+const AverageRating = ({ rating }) => {
+  let average = 0;
+  let count = 0;
+
+  if (rating && rating.count) {
+    average = rating.average;
+    count = rating.count;
+  }
+
+  return (
+    <div className={style.center}>
+      <RatingStars value={average} display="large" />
+      <RatingCount count={count} />
```

Comment:
Since we already know the rating count here we can choose to not render the component if the count is 0. This will skip the rendering process of that component even if there is no output.
`{count > 0 && <RatingCount count={count} />}`

Reply:
Done.